projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -1236,17 +1236,19 @@ private boolean dataPlaneDependenciesExist(TestrigSettings testrigSettings) {
             namesByPath.size());
     namesByPath.forEach(
         (inputPath, name) -> {
-          logger.debug(
-              "Reading and gunzipping: "
-                  + outputClassName
-                  + " '"
-                  + name
-                  + "' from '"
-                  + inputPath
-                  + "'");
-          byte[] data = fromGzipFile(inputPath);
-          logger.debug(" ...OK\n");
-          dataByName.put(name, data);
+          if (!Files.isDirectory(inputPath)) {
```

Comment:
We have an open issue to scan directories recursively for input files. This filtering would then not be necessary, as the inputPath args supplied would be guaranteed not to be directories.